AppCenterAuth/AppCenterAuth/Internals/MSAuthPrivate.h

```diff
@@ -46,6 +46,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(atomic, nullable) MSSignInCompletionHandler signInCompletionHandler;
 
+/**
+ * Completion handler for refresh completion.
+ */
+@property(atomic, nullable) MSSignInCompletionHandler refreshCompletionHandler;
```

Comment:
nit: refreshCompletionHandler is not a signIn completion handler. It would be better to say MSAcquireTokenCompletionHandler for internal handler by typedef-ing it?